Week4/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/MovieDetailViewHolder.kt

```diff
@@ -0,0 +1,14 @@
+package com.raywenderlich.myfavoritemovies
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.movie_detail_view_holder.view.*
+
+class MovieDetailViewHolder(movieView: View) : RecyclerView.ViewHolder(movieView) {
+    val moviePosterImageView: ImageView = movieView.moviePosterImageView
```

Comment:
If you use the `setData()` function approach, as I've mentioned, you don't have to assign these variables to expose them, you can just pick them up within the `setData()` function!